Twitter_API/code.py

```diff
@@ -0,0 +1,133 @@
+"""
+Halloween countdown for PyPortal.
+
+Adafruit invests time and resources providing this open source code.
+Please support Adafruit and open source hardware by purchasing
+products from Adafruit!
+
+Written by Dave Astels for Adafruit Industries
+Copyright (c) 2019 Adafruit Industries
+Licensed under the MIT license.
+
+All text above must be included in any redistribution.
+"""
+
+#pylint:disable=invalid-name
+import time
+import binascii
+import json
+import board
+import busio
+from adafruit_pyportal import PyPortal
+from digitalio import DigitalInOut
+from adafruit_esp32spi import adafruit_esp32spi
+import adafruit_esp32spi.adafruit_esp32spi_requests as requests
+import adafruit_logging as logging
+
+username = 'codewisdom'
+
+logger = logging.getLogger('main')
```

Comment:
i know you like logger stuff but lets keep it like the other pyportal demos and stick to print() :)